logbook-spring-boot-starter/src/test/java/org/zalando/logbook/spring/ExcludeTest.java

```diff
@@ -0,0 +1,97 @@
+package org.zalando.logbook.spring;
+
+/*
+ * #%L
+ * Logbook: Spring
+ * %%
+ * Copyright (C) 2015 Zalando SE
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
+import org.zalando.logbook.Correlator;
+import org.zalando.logbook.Logbook;
+import org.zalando.logbook.MockRawHttpRequest;
+import org.zalando.logbook.RawHttpRequest;
+
+import java.io.IOException;
+import java.util.Optional;
+import java.util.function.Predicate;
+
+import static java.util.Optional.empty;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@ContextConfiguration
+@TestPropertySource(properties = "spring.config.name = exclude")
+public final class ExcludeTest extends AbstractTest {
+
+    @Configuration
+    public static class TestConfiguration {
+
+        @Bean
+        public Logger httpLogger() {
+            final Logger logger = mock(Logger.class);
+            when(logger.isTraceEnabled()).thenReturn(true);
+            return logger;
+        }
+        
+        @Bean
+        public Predicate<RawHttpRequest> requestPredicate() {
+            return request -> 
+                    !"/health".equals(request.getRequestUri());
```

Comment:
> Are we just talking about the Spring Starter or Logbook in general?

Right now I'm using the ant-style path matcher only inside the starter module which is fine, in terms of dependency management. But there are a couple of generic predicates already in the core (not part of this PR, since already merged), see `RequestPredicates`. Those match on string equals and Pattern (regex). I'd like to only support one of those, and I prefer the ant-style.

> If an app needs a special request predicate, it can come up best with it's own strategies how to match requests. If that involves more complex URL analysis, the app can do what fits best.

I'd consider URL path matching predicates universally useful to all our clients, so I see the use case for having something like it in the logbook *ecosystem*.